Makefile

```diff
@@ -40,9 +47,14 @@ uninstall: manifests
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests
-	cd config/manager && kustomize edit set image controller=${IMG}
+	cd config/manager && kustomize edit set image controller=${CONTROLLER_IMG}
 	kustomize build config/default | kubectl apply -f -
 
+# Deploy cleanup_agent in the configured Kubernetes cluster in ~/.kube/config
+deploy-cleanup-agent: manifests
+	cd config/cleanup_agent && kustomize edit set image cleanup_agent=${CLEAN_IMG}
+	kustomize build config/cleanup_agent | kubectl apply -f -
+
```

Comment:
If we have the two together in the same target, do we always deploy the controller and clean-up agent together? I think so, just want to learn about what you think.

Reply:
I have added the target, I think it would be nice to have a target that we could deploy both at the same time but not necessarily have to use it every time.